validation/schema.json

```diff
@@ -218,6 +218,45 @@
         ]
       }
     },
+    "credits": {
+      "type": "array",
+      "items": {
+        "type": "object",
+        "properties": {
+          "name": {
+            "type": "string"
+          },
+          "contact": {
+            "type": "array",
+            "items": {
+              "type": "object",
+              "properties": {
+                "type": {
+                  "type": "string",
+                  "enum": [
+                    "EMAIL",
+                    "GITHUB",
+                    "OTHER",
+                    "TWITTER",
+                    "URL"
```

Comment:
These are just the types that came off the top of my head. Happy to expand this list with any other initial values that you think are worth including.